Taskfile.yaml

```diff
@@ -67,7 +66,7 @@ tasks:
       - task: generate-sqlc
   seed-dev:
     cmds:
-      - sh ./hack/dev/run-go-with-env.sh run github.com/steebchen/prisma-client-go migrate deploy
+      - sh ./hack/dev/run-go-with-env.sh run github.com/steebchen/prisma-client-go migrate dev --skip-generate
```

Comment:
I would prefer not to do this as otherwise this will ask you to create a migration which is then however not synced with atlas. Why do you need to run dev here? Alternatively we can also do db push?

Reply:
Without this, I believe the prisma command which is generating `schema.sql` wasn't generating the correct `schema.sql` as it looks at the migration directory, which meant the atlas migrations weren't being created properly. On the other hand, if we actually deploy from the prisma schema using prisma, the atlas migrations won't succeed. This is the only configuration that seems to account for both cases (both @grutt and I had issues with migrations until we reverted).

Reply:
hmm I see, I had issues the other way round but as long as it works for you now that's fine then